spec/lib/pact_broker/api/decorators/webhook_decorator_spec.rb

```diff
@@ -131,6 +131,80 @@ module Decorators
               expect(parsed_json[:request][:headers][:'Authorization']).to eq "**********"
             end
           end
+
```

Comment:
All these tests now belong in spec/lib/pact_broker/webhooks/url_redactor_spec.rb - the only thing that belongs in the webhook_decorator_spec.rb is asserting that the redactor gets called with the right arguments.

Reply:
Sorry, updated tests to reflect new functionality but didn't move them into a corresponding named spec file, will sort and resubmit!

Reply:
tests moved into `spec/lib/pact_broker/webhooks/url_redactor_spec.rb`